lib/ohai/plugins/scsi.rb

```diff
@@ -0,0 +1,51 @@
+#
+# Author:: Phil Dibowitz <phil@ipom.com>
+# Copyright:: Copyright (c) 2018 Facebook, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Ohai.plugin(:Lsscsi) do
+  depends "platform"
+  provides "scsi"
+
+  require "mixlib/shellout"
+
+  collect_data(:linux) do
+    devices = Mash.new
+    lsscsi = shell_out("lsscsi")
```

Comment:
Should probably either use `shell_out!` or have an explicit `return if lsscsi.error?` so we don't try and parse any potential error output.